pkg/remote/aws/iam_role_policy_attachment_details_fetcher.go

```diff
@@ -0,0 +1,39 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+)
+
+type IamRolePolicyAttachmentDetailsFetcher struct {
+	reader       terraform.ResourceReader
+	deserializer *resource.Deserializer
+}
+
+func NewIamRolePolicyAttachmentDetailsFetcher(provider terraform.ResourceReader, deserializer *resource.Deserializer) *IamRolePolicyAttachmentDetailsFetcher {
+	return &IamRolePolicyAttachmentDetailsFetcher{
+		reader:       provider,
+		deserializer: deserializer,
+	}
+}
+
+func (r *IamRolePolicyAttachmentDetailsFetcher) ReadDetails(res resource.Resource) (resource.Resource, error) {
+	ctyVal, err := r.reader.ReadResource(terraform.ReadResourceArgs{
+		Ty: aws.AwsIamRolePolicyAttachmentResourceType,
```

Comment:
I know we already did that in every other details fetchers, but is there a reason to not use the resource type defined by the enumerator ? We could just call `res.TerraformType()` here